spec/services/account_search_service_spec.rb

```diff
@@ -72,7 +72,7 @@
         describe 'and there is no account provided' do
           it 'uses search_for to find matches' do
             allow(Account).to receive(:search_for)
-            subject.call('two@example.com', 10, false, nil)
+            subject.call('two@example.com', 10, nil, resolve: false)
```

Comment:
https://github.com/tootsuite/mastodon/blob/82884ac5c43092c0b0625f7d81c930bb023c9469/app/services/account_search_service.rb#L6

I feel that the thing before change is correct. Am I looking at the right place?

Reply:
@ykzts indeed, you're looking at master, not at feature-lists-ui (on top of which this PR applies): https://github.com/tootsuite/mastodon/blob/feature-lists-ui/app/services/account_search_service.rb#L6

Reply:
Oh sorry. I misunderstood that this is for master branch.